sdk/core/azure-core-http-okhttp/src/main/java/module-info.java

```diff
@@ -1,16 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-module azure.core.http.okhttp {
+module com.azure.core.http.okhttp {
 
     requires transitive com.azure.core;
+    requires com.azure.core.test;
```

Comment:
Do we want to make `com.azure.core.test` `requires transitive`?

Reply:
I'm curious why we would want to make it transitive? I thought `transitive` is if you wanted anyone consuming azconfig to implicitly reads azure-core-test. I don't think we ship anything publicly that requires this?

Reply:
I don't think anyone wanting to use okhttp module would also want to use test module. If required we can update later.